src/universal/modules/welcome/containers/Welcome/Welcome.js

```diff
@@ -110,14 +147,28 @@ export default class WelcomeContainer extends Component {
     dispatch(push(`/team/${teamId}`));
   };
 
+  nextPage() {
+    this.setState({ page: this.state.page + 1 });
```

Comment:
i kinda wanna hold this in redux since in the future, other things might care about what part of the wizard we're in.